plans/phase-1/final-cr-definition.md

```diff
@@ -0,0 +1,845 @@
+  multiorchAvailable: "True"
+  ```
+
+  ## TableGroup CR - Read-Only Child of MultigresCluster
+
+```yaml
+# This child CR is created from the 'orders_tg' entry
+# under 'production_db' in the MultigresCluster CR.
+#
+# The MultigresCluster controller does the following:
+# 1. Copies the *relevant* images (postgres, multipooler) into `spec.images`.
+# 2. The multigresCluster controller watches and resolves the `deploymentTemplate` entries into the 'shardTemplate'.
+#
+apiVersion: multigres.com/v1alpha1
+kind: TableGroup
+metadata:
+  # Name is derived from database + table group name
+  name: "production-db-orders-tg"
+  namespace: multigres
+  creationTimestamp: "2025-10-21T10:30:02Z"
+  generation: 1
+  resourceVersion: "12347"
+  uid: "d4e5f6a7-1234-5678-90ab-f0e1d2c3b4a7"
+  labels:
+    multigres.com/cluster: "example-multigres-cluster"
+    multigres.com/database: "production_db"
+    multigres.com/table-group: "orders_tg"
+  ownerReferences:
+  - apiVersion: multigres.com/v1alpha1
+    kind: MultigresCluster
+    name: "example-multigres-cluster"
+    uid: "a1b2c3d4-1234-5678-90ab-f0e1d2c3b4a5"
+    controller: true
+    blockOwnerDeletion: true
+spec:
+  # The parent MultigresCluster controller passes down the
+  # images relevant to this CR's children (Shard).
+  images:
+    multipooler: "multigres/multigres:latest"
+    postgres: "postgres:15.3"
+
+  partitioning:
+    shards: 2
+
+# The multigresCluster controller watches and resolves the `deploymentTemplate` entries into the 'shardTemplate'.
+  shardTemplate:
+    pools:
+      - type: "replica"
+        cell: "us-west-2"
+        replicas: 2
+        dataVolumeClaimTemplate:
+          accessModes: ["ReadWriteOnce"]
+          resources:
+            requests:
+              storage: "500Gi"
+        postgres:
+          resources:
+            requests:
+              cpu: "4"
+              memory: "8Gi"
+        multipooler:
+          resources:
+            requests:
+              cpu: "1"
+              memory: "512Mi"
+
+      - type: "readOnly"
+        cell: "us-east-1"
+        replicas: 1
+        dataVolumeClaimTemplate:
+          accessModes: ["ReadWriteOnce"]
+          resources:
+            requests:
+              storage: "500Gi"
+        postgres:
+          resources:
+            requests:
+              cpu: "2"
+              memory: "4Gi"
+        multipooler:
+          resources:
+            requests:
+              cpu: "500m"
+              memory: "256Mi"
+            limits:
+              cpu: "1"
+              memory: "512Mi"
+status:
+  conditions:
+  - type: Available
+    status: "True"
+    lastTransitionTime: "2025-10-21T10:37:00Z"
+    message: "All shards are healthy"
+  shards: 2
+  readyShards: 2
+  ```
+
+  ## Shard CR - Read-Only Child of TableGroup
+
+```yaml
+# This child CR is created by the 'TableGroup' controller
+# for 'production-db-orders-tg'. This is shard "0" of 2.
```

Comment:
The phrase of "shard "0" of 2" was quite confusing. I believe this is about the first shard (idx 0) out of 2 shards?